src/cpp/rtps/reader/StatefulReader.cpp

```diff
@@ -1472,15 +1452,7 @@ bool StatefulReader::begin_sample_access_nts(
 
 void StatefulReader::end_sample_access_nts(
         CacheChange_t* change,
-        WriterProxy*& wp,
-        bool mark_as_read)
-{
-    change_read_by_user(change, wp, mark_as_read);
-}
-
-void StatefulReader::change_read_by_user(
```

Comment:
This method is referenced in a comment, and also needs to be removed from the RTPSReader mock